webpack/assets/javascripts/react_app/components/Editor/Editor.js

```diff
@@ -188,7 +188,9 @@ class Editor extends React.Component {
           isMasked={isMasked}
           isRendering={isRendering}
         />
-        <textarea className="hidden" name={name} value={value} readOnly />
+        {!readOnly && (
+          <textarea className="hidden" name={name} value={value} readOnly />
```

Comment:
isn't the `readOnly` prop useless now as it is always false?

Reply:
its a `textarea` without `onChange` handler. due to the Editor taking care of the `onChange`, it needs to be `readOnly` aswell.

Reply:
ack! thanks for explanation! :heart:

Reply:
does this warrant adding a test?

Reply:
added a test. @ohadlevy